### PR TITLE
updated colors for parameters and keywords in PyCharm

### PR DESCRIPTION
**Pull request by pr0grammr**

### CODE REVIEW

File: resources/TrashPandaTheme.icls [hunk 1]
```diff
@@ -1,11 +1,11 @@
-<scheme name="TrashPandaTheme" version="142" parent_scheme="Darcula">
+<scheme name="TrashPandaTheme copy" version="142" parent_scheme="Darcula">
```
Comment (jasonhulbert) [inline resources/TrashPandaTheme.icls]: Remove "copy" from scheme name

File: resources/TrashPandaTheme.icls [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
   <option name="FONT_SCALE" value="1.0" />
   <metaInfo>
```
Comment (jasonhulbert) [inline resources/TrashPandaTheme.icls]: Remove the whole `metaInfo` block

File: resources/TrashPandaTheme.icls [hunk 3]
```diff
@@ -714,15 +701,15 @@
     <option name="XML_TAG" baseAttributes="DEFAULT_TAG" />
     <option name="XML_TAG_DATA" baseAttributes="TEXT" />
     <option name="XML_TAG_NAME" baseAttributes="DEFAULT_KEYWORD" />
-    <option name="XPATH.KEYWORD" baseAttributes="DEFAULT_KEYWORD" />
+    <option name="XPATH.KEYWORD" baseAttributes="" />
```
Comment (jasonhulbert) [inline resources/TrashPandaTheme.icls]: `XPATH.KEYWORD` should have `baseAttributes="DEFAULT_KEYWORD"` to inherit the base keyword appearance

File: resources/TrashPandaTheme.icls [hunk 4]
```diff
@@ -714,15 +701,15 @@
...
     <option name="XPATH.XPATH_NAME">
       <value>
         <option name="FOREGROUND" value="c6ccd3" />
         <option name="BACKGROUND" value="222629" />
         <option name="EFFECT_TYPE" value="5" />
       </value>
     </option>
-    <option name="XPATH.XPATH_VARIABLE" baseAttributes="DEFAULT_LOCAL_VARIABLE" />
+    <option name="XPATH.XPATH_VARIABLE" baseAttributes="" />
```
Comment (jasonhulbert) [inline resources/TrashPandaTheme.icls]: `XPATH.XPATH_VARIABLE` should have `baseAttributes="DEFAULT_LOCAL_VARIABLE"` to inherit the base local appearance

File: resources/TrashPandaTheme.icls [hunk 5]
```diff
@@ -586,24 +578,19 @@
         <option name="FOREGROUND" value="6a7175" />
       </value>
     </option>
-    <option name="REGEXP.BRACES">
```
Comment (jasonhulbert) [inline resources/TrashPandaTheme.icls]: It looks like the regex updates that were updated in the last release are being modified here. I didn't start up PyCharm but these changes are affecting the "RegExp" color scheme settings generally.

**Comment by jasonhulbert** [comment]
@pr0grammr I'd like to get your changes in so I'll just make the needed updates in PyCharm and get them out in the publish.

**Comment by pr0grammr** [comment]
@jasonhulbert i can make the changes :) i was a bit busy the last days 🙈

**Comment by jasonhulbert** [comment]
@pr0grammr I'm publishing an update today (v1.5). I think the regex changes you were hoping to see will be resolved in that update. If not, we can reopen this. Thanks!

**Pull request closed without merging**